puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/vars/hammer-cli-master-release.groovy

```diff
@@ -0,0 +1,6 @@
+def git_url = 'https://github.com/theforeman/hammer-cli.git'
```

Comment:
Side note: I've been wondering if we could use the github URL from the job properties here but for now I'm fine with hardcoding this.

Reply:
i thought hard about this too a few months back, then decided it wasn't worth all the clever string manipulation to avoid adding two extra variables.

Reply:
Out of interest: what string manipulation? You can `git clone https://github.com/theforeman/hammer-cli` so the URL is exactly the same.